services/s3_gate/cfg/config.yml

```diff
@@ -0,0 +1,19 @@
+logger:
+  level: debug
+  
+# Timeout to check node health during rebalance
+healthcheck_timeout: 15s
```

Comment:
Is `rebalance_interval` dropped intentionally?

Reply:
no, added back